tcms/testcases/forms.py

```diff
@@ -338,21 +339,25 @@ class QuickSearchCaseForm(forms.Form):
 
     def clean_case_id_set(self):
         case_id_set = self.cleaned_data['case_id_set']
-        if case_id_set:
-            try:
-                return [int(cid) for cid in set(case_id_set.split(','))]
-            except ValueError:
-                raise forms.ValidationError('Please input valid case id(s). '
-                                            'use comma to split more than one '
-                                            'case id. e.g. "111, 222"')
-        else:
-            raise forms.ValidationError('Please input valid case id(s). '
-                                        'use comma to split more than one '
-                                        'case id. e.g. "111, 222"')
 
+        if not case_id_set:
+            raise forms.ValidationError(VALIDATION_ERROR_MESSAGE)
+        try:
+            case_ids = []
+            for case_id in case_id_set.split(','):
+                case_ids.append(int(case_id))
+            return case_ids
+        except ValueError:
+            raise forms.ValidationError(VALIDATION_ERROR_MESSAGE)
+
+
+VALIDATION_ERROR_MESSAGE = 'Please input valid case id(s). ' \
```

Comment:
Use `ugettext_lazy()` to mark this as translatable and also update translation strings by `makemessages`

Reply:
Also it is better to define constants before the function that uses them, or even better at the top of the module so thay can easily be found.

Reply:
Done, althought a lot of lines appears to have changes in the translations files. Please verify it all is ok, or I will drop the commit

Reply:
NVM, there were conflicts, which were resolved. Should be fine now